### PR TITLE
ICP-5559 change orvibo motion sensor to zigbee motion detector

### PR DESCRIPTION
**Pull request by jinkangzhangS**
1.change orvibo motion sensor to zigbee motion detector
2.add battery configurations
3.change runLocally and executeCommandsLocally to true

### CODE REVIEW

**Comment by jinkangzhangS** [comment]
@greens @tpmanley

File: devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy [hunk 1]
```diff
@@ -18,14 +18,14 @@
 import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
 import physicalgraph.zigbee.zcl.DataType
 metadata {
-	definition(name: "Orvibo Motion Sensor", namespace: "smartthings", author: "SmartThings", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, mnmn: "SmartThings", vid: "generic-motion-2") {
+	definition(name: "zigbee-motion-detector", namespace: "smartthings", author: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: true, mnmn: "SmartThings", vid: "generic-motion-2") {
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy]: Our naming convention is to call this `Zigbee Motion Detector`. In the past the `B` would be capitalized but the Zigbee Alliance changed the branding rules recently so now the `b` is lowercase.

File: devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy [hunk 2]
```diff
@@ -137,4 +137,5 @@ def refresh() {
 def configure() {
 	log.debug "configure"
 	sendEvent(name: "checkInterval", value:20 * 60 + 2*60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	return zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 21600, 0x10)
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy]: The commands sent here should be:
* read battery percentage
* send the enrollResponse to enroll the device in the IAS Zone
* configure battery reporting
* read the IAS Zone status

File: devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy (hunk 1 above)
Comment (varzac) [reply devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy]: `minHubCoreVersion` should be `000.018.0000` instead as this wouldn't be supported on 17.

**Review by greens — APPROVED** [review]
@tpmanley looks to me like they took care of your requests, but I'll give you the final say.

**Review by tpmanley — APPROVED** [review]

File: devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy [hunk 3]
```diff
@@ -18,14 +18,14 @@
...
-	definition(name: "Orvibo Motion Sensor", namespace: "smartthings", author: "SmartThings", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false, mnmn: "SmartThings", vid: "generic-motion-2") {
+	definition(name: "Zigbee Motion Detector", namespace: "smartthings", author: "SmartThings", runLocally: true, minHubCoreVersion: '000.018.0000', executeCommandsLocally: true, mnmn: "SmartThings", vid: "generic-motion-2") {
 		capability "Motion Sensor"
 		capability "Configuration"
 		capability "Battery"
 		capability "Refresh"
 		capability "Health Check"
 		capability "Sensor"
-		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500,0001", manufacturer: "ORVIBO",model:"895a2d80097f4ae2b2d40500d5e03dcc"
+		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0003,0500,0001", model:"895a2d80097f4ae2b2d40500d5e03dcc", deviceJoinName: "Orvibo Motion Sensor"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy:28]: @jinkangzhangS does this device have contents in the Manufacturer attribute? There is some concern the device won't be fingerprinted correctly if the fingerprint doesn't have both `manufacturer` and `model`.

**Pull request merged**